gensim/models/word2vec.py

```diff
@@ -1869,3 +1873,15 @@ def __iter__(self):
         model.accuracy(args.accuracy)
 
     logger.info("finished running %s", program)
+    
+def methodize(func, instance):
```

Comment:
This was done on purpose. Please see [here](http://stackoverflow.com/a/861109) for reference.

`MethodType` binds the method to the calling instance. What I'm doing here is something like overloading `load` based on whether it is called on instance or class.

So the argument `load` to `methodize` is the name of the function that is to be bound.